commands/latency-latest.md

```diff
@@ -6,6 +6,8 @@ Each reported event has the following fields:
 * Unix timestamp of the latest latency spike for the event.
 * Latest event latency in millisecond.
 * All-time maximum latency for this event.
+* All-time summary latency for this event, added in 8.1.
+* The event count that trigger the latency, added in 8.1.
```

Comment:
Sum != summary :smile:

"trigger the latency" sounds weird.

I found better words "latency spike" and "time series" on this page: https://valkey.io/topics/latency-monitor/

```suggestion
* Sum of the latencies recorded in the time series for this event, added in 8.1.
* The number of latency spikes recorded in the time series for this event, added in 8.1.
```

Reply:
> Sum != summary

haha, i did a search for it, seem like a bad search result.